src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -215,6 +221,12 @@ export function CollapsibleNav({
       outsideClickCloses={false}
     >
       <EuiFlexItem className="eui-yScroll">
+        <CollapsibleNavHeader
+          getUrlForApp={getUrlForApp}
+          workspaces={workspaces}
+          basePath={basePath}
+        />
```

Comment:
I suppose we can use `useOpensearchDashboards` to get all the props that is needed in CollapsivleNavHeader. In that way CollapsibleNav do not have to depend on these props.